### PR TITLE
[table] fix wrapText on EditableCell component

### LINKED ISSUE #2700: wrapText has no effect on EditableCell
#### Environment

- __Package version(s)__: 3.0.0
- __Browser and OS versions__: Chrome 67.0.3396.99 MacOS 10.13.6

#### Steps to reproduce

1. add the prop `wrapText={true}` to the _EditableCell_ component

#### Actual behavior

The text does not wrap

#### Expected behavior

The text to wrap

#### Possible solution

add wrapText attribute to the child Cell component: PR https://github.com/palantir/blueprint/pull/2701

### PR DESCRIPTION
**Pull request by face**
#### Checklist

- [ ] [Enable CircleCI for your fork](https://circleci.com/add-projects)  (I tried to do this, but circecli is not showing my fork, and there is an error that github service is partial degregated)
- [x] Include tests
- [ ] Update documentation

#### Changes proposed in this pull request:

Fixes bug where wrapText={true} has doesn't work for 

#### Reviewers should focus on:

fix and tests

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @face! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/table/test/editableCellTests.tsx [hunk 1]
```diff
@@ -126,4 +126,16 @@ describe("<EditableCell>", () => {
...
         expect(onChangeSpy.firstCall.args).to.deep.equal([CHANGED_VALUE, ROW_INDEX, COLUMN_INDEX]);
     });
+
+    it("defaults to no wrapText", () => {
+        const elem = mount(<EditableCell />);
+
+        expect(elem.find(`.${Classes.TABLE_NO_WRAP_TEXT} div`).length).to.equal(2);
+    });
+
+    it("wraps text when wrapsText is true", () => {
```
Comment (giladgray) [inline packages/table/test/editableCellTests.tsx]: wrapText

File: packages/table/test/editableCellTests.tsx [hunk 2]
```diff
@@ -126,4 +126,16 @@ describe("<EditableCell>", () => {
...
+        const elem = mount(<EditableCell wrapText={true} />);
+
+        expect(elem.find(`.${Classes.TABLE_NO_WRAP_TEXT} div`).length).to.equal(0);
```
Comment (giladgray) [inline packages/table/test/editableCellTests.tsx]: This is an inscrutable selector. Why the child div? Is it necessary? Can you come up with a more legible assertion? Maybe just the length of children on TABLE_NO_WRAP_TEXT?

File: packages/table/test/editableCellTests.tsx (hunk 1 above)
Comment (face) [reply packages/table/test/editableCellTests.tsx]: fixed.

File: packages/table/test/editableCellTests.tsx (hunk 2 above)
Comment (face) [reply packages/table/test/editableCellTests.tsx]: Thank you @giladgray.   You are correct, the div is not necessary.   Added children().length.  Is this what you  wanted?

File: packages/table/test/editableCellTests.tsx [hunk 3]
```diff
@@ -126,4 +126,16 @@ describe("<EditableCell>", () => {
         elem.find("input").simulate("blur");
         expect(onChangeSpy.firstCall.args).to.deep.equal([CHANGED_VALUE, ROW_INDEX, COLUMN_INDEX]);
     });
+
+    it("defaults to no wrapText", () => {
+        const elem = mount(<EditableCell />);
+
+        expect(elem.find(`.${Classes.TABLE_NO_WRAP_TEXT}`).children().length).to.equal(1);
```
Comment (giladgray) [inline packages/table/test/editableCellTests.tsx]: actually this doesn't do what you think it does: https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#children-now-has-slightly-different-meaning. 

i think this test can be simply `expect(elem.find(`.${Classes.TABLE_NO_WRAP_TEXT}`).exists()).to.be.true;` and the other one `.to.be.false`

Comment (giladgray) [reply packages/table/test/editableCellTests.tsx]: the `.exists()` check is very intuitive: no wrap text means the NO_WRAP_TEXT element exists.

Comment (giladgray) [reply packages/table/test/editableCellTests.tsx]: @face can I get a push from you real quick to resolve this?

**Comment by giladgray** [comment]
merging, will fix comments myself

**Comment by face** [comment]
thanks @giladgray, sorry I dropped the ball and didn't see your request.  I got pulled onto another project and am just now back on the one using bluprintjs.     I need to figure out a better way to manage my github alerts as I would have addressed this had I not missed your request.   thanks again for finalizing and merging the fix.

**Comment by giladgray** [comment]
Sure thing @face! It was a good fix, thanks for the contribution.

>

**Pull request merged**